xonsh/aliases.py

```diff
@@ -208,6 +208,7 @@ def __init__(self, f, acc_args=()):
         """
         self.f = f
         self.acc_args = acc_args
+        self.__name__ = self.__class__.__name__
```

Comment:
Should this be `self.__name__ = getattr(f, "__name__", self.__class__.__name__)` instead? That is, should it take the function name?